code/unit_tests/chemistry_tests.dm

```diff
@@ -12,13 +12,11 @@ datum/unit_test/specific_heat/start_test()
 
 	var/error_count = 0
 
-	for(var/type_found in subtypesof(/datum/reagent/))
-		var/datum/reagent/R = new type_found
-		if(istype(R) && R.name)
+	for(var/decl/reagent/R in decls_repository.get_decls_of_subtype(/decl/reagent/))
+		if(R.name)
```

Comment:
Looks like someone removed the reagent name from the log messages, since this condition is pointless?

Either way, we talked about making sure every reagent has a name. So it isn't needed.